server/channels/app/platform/config.go

```diff
@@ -326,12 +326,20 @@ func (ps *PlatformService) LimitedClientConfig() map[string]string {
 }
 
 func (ps *PlatformService) IsFirstUserAccount() bool {
-	count, err := ps.Store.User().Count(model.UserCountOptions{IncludeDeleted: true})
-	if err != nil {
-		return false
+	if ps.fetchUserCountForFirstUserAccountCheck.Load() {
```

Comment:
A thundering herd may still hit this API many times before the flag gets set to false.  Probably not huge deal for now (let's get the fix out) but something that blocks on first pass (i.e. sync.Once) would truly ensure it gets called exactly once no matter how many concurrent users log in or reconnect.

Reply:
Created https://mattermost.atlassian.net/browse/MM-52973 and submitted a fix via https://github.com/mattermost/mattermost-server/pull/23549